src/MacVim/MMFullScreenWindow.m

```diff
@@ -104,7 +104,24 @@ - (MMFullScreenWindow *)initWithWindow:(NSWindow *)t view:(MMVimView *)v
 
     // NOTE: Vim needs to process mouse moved events, so enable them here.
     [self setAcceptsMouseMovedEvents:YES];
-
+  
+    // Prefer to get the fade time from preferences, but default to the original time
+    // if the key doesn't exist yet.
+    NSUserDefaults * defaults = [NSUserDefaults standardUserDefaults];
+    if ([[[defaults dictionaryRepresentation] allKeys] containsObject:MMFullScreenFadeTimeKey]) {
```

Comment:
I think we can set default value in https://github.com/macvim-dev/macvim/blob/master/src/MacVim/MMAppController.m#L185-L219?